package-lock.json

```diff
@@ -455,6 +455,12 @@
 				"semver": "^6.2.0"
```

Comment:
Can you remove the changes to this file?

Reply:
Done :)

Reply:
Hi ! 
Just updated this PR with the most recent changes from master.
(I also removed the changes from package-lock.json, now I get no diffs)